docs/get-started/deploy-cloudfoundry-for-enterprise.md

```diff
@@ -0,0 +1,129 @@
+
+If the number of disks in one storage account exceeds the maximum number, you need to split these disks into multiple storage accounts for performance optimizations.
+
+### How
+
+1. Make a plan for the mapping between the resource pools and the jobs.
+
+  For example, if your deployments need 80 runners (DEAs), then the number of disks is at least 80 which exceeds the maximum number in one storage account. So you need to configure at least 2 resource pools for runners.
+
+2. Specify an additional storage account in each of the resource pools.
+
+  Open your manifest, and configure `storage_account_name: <new-storage-account>` in your resource pool as follows. Then all the VMs in the pool will store their disks in the storage account `<new-storage-account>`.
+
+  ```
+  - name: resource_router
+    network: default
+    stemcell:
+      name: bosh-azure-hyperv-ubuntu-trusty-go_agent
+      version: '0000'
+    cloud_properties:
+      storage_account_name: <new-storage-account>
+      instance_type: Standard_D1
+  ```
+
+3. Create new storage accounts which are configured in `resource_pools` manually if they do not exist.
+
+  Create `<new-storage-account-name>`.
+
+  ```
+  azure storage account create --location <location> --type <account-type> --resource-group <resource-group-name> <new-storage-account-name>
+  ```
+
+  Create storage containers `bosh` and `stemcell` in `<new-storage-account-name>`.
+
+  ```
+  azure storage account keys list --resource-group <resource-group-name> <new-storage-account-name> # Get your storage account key
+  azure storage container create --account-name <new-storage-account-name> --account-key <storage-account-key> bosh
+  azure storage container create --account-name <new-storage-account-name> --account-key <storage-account-key> stemcell
+  ```
+
+  For more information about how to create storage accounts and storage containers, click [**HERE**](./deploy-bosh-manually.md#setup-a-default-storage-account).
+
+4. Deploy your Cloud Foundry.
+
+## Example deployment
+
+We provide an example manifest to deploy Cloud Foundry for enterprise.
+
+* [**cf-for-enterprise-224.yml**](../example_manifests/cf-for-enterprise-224.yml)
+
+In the above example manifest, there is a specific resource pool for each job. An availability set and a unique storage account name are specified in the resource pool. After the deployment, all the VMs in the pool will be put into the availability set, and their disks are stored in the storage account.
+
+The deployment includes 21 VMs and additional 13 storage accounts.
+
+```
```

Comment:
Below is a sample manifest file for deploy Cloud Foundry with availability set and multiple storage accounts. The deployment includes a typical scenario with 21 VMs and additional 13 storage accounts; there is a specific resource pool for each job. An availability set and a unique storage account are specified for each resource pool, and multiple resource pools for the "runner" pool due to its disk needs.  You can adjust the number of resource pools based on this sample file.